config/locales/en_AU.yml

```diff
@@ -628,7 +628,7 @@ en_AU:
         no_name: No name
         blank_enterprise: some products do not have an enterprise defined
         reset_absent?: Reset absent products
-        reset_absent_tip: Set stock to zero for all exiting products not present in the file
+        reset_absent_tip: Set stock to zero for all existing products not present in the file
```

Comment:
All changes in locales other than en.yml will be overridden with the next release. Transifex is our source of truth. I hope that it will be updated automatically there.